packages/@lwc/integration-karma/test/mixed-shadow-mode/retargeting/x/syntheticParent/syntheticParent.html

```diff
@@ -0,0 +1,9 @@
+<template>
+    <x-native-child>
+        <button class="lwc-slotted-button"></button>
+    </x-native-child>
```

Comment:
What happens for a synthetic child?

Reply:
The existing `isSyntheticShadowRoot()` check accounts for a synthetic child, so I needed to add an additional check for native. I'll add a test for this for completeness.